assets/src/edit-story/components/inspector/inspectorProvider.js

```diff
@@ -59,6 +59,17 @@ function InspectorProvider({ children }) {
     500
   );
 
+  const { tab: highlightedTab } = useHighlights(({ tab }) => ({ tab }));
+
+  useEffect(() => {
+    if (highlightedTab === 'document') {
```

Comment:
Could these constants be moved somewhere central so that it's just calling `setTab(highlightedTab)`?